main.c

```diff
@@ -1,4 +1,7 @@
 #include <stdio.h>
 
 int main(){
+  printf("hello world");
```

Comment:
```suggestion
  printf("Hello world\n");
```
最初の文字は大文字にしましょう。
改行をしましょう。